src/collections/_documentation/platforms/index.md

```diff
@@ -19,6 +19,7 @@ If your use-case is very specific, or not covered by Sentry, you'll find documen
 
 These SDKs are [maintained and supported](https://forum.sentry.io) by the Sentry community. While generally our community does a great job at responding to issues, it's important to understand that Sentry staff cannot help you with issues using a community-supported SDK
 
+* [ Flutter ](https://flutter.dev/docs/cookbook/maintenance/error-reporting)
```

Comment:
```suggestion
* [_Flutter_](https://flutter.dev/docs/cookbook/maintenance/error-reporting)
```

@eurica The link should probably be italicized as with the rest of the community SDK links.